rails-snow-queen/app/assets/javascripts/quotingPanel.js

```diff
@@ -54,7 +59,9 @@ class QuotingPanel {
     const areaNode = document.getElementById("calculatedArea");
     areaNode.innerText = `${this.totalAreaInSqFt.toLocaleString(undefined, {maximumFractionDigits: 0})}`;
     areaNode.parentNode.parentNode.classList.remove("hidden");
-    
```

Comment:
Strictly speaking, L61 should _not_ be part of `updateAreaNode`, it should probably go in its own `showHiddenFields` or something like that (and probably deserves an ID on the container so that you aren't as tied to the DOM structure (like you are now with `areaNode.parentNode.parentNode`).

Reply:
done